rct229/rulesets/ashrae9012019/section22/section22rule20.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
             each_rule=Section22Rule20.HeatRejectionRule(),
             index_rmd=BASELINE_0,
             id="22-20",
-            description="The baseline minimum condenser water reset temperature is per Table G3.1.3.11.",
+            description="The baseline heat rejection device leaving water temperature shall be modeled as specified in Table G3.1.3.11.",
```

Comment:
I think this should also be where weather permits

Reply:
I agree it seems silly to have them as separate rules. Right now 22-19 checks the method and 22-20 checks the temperature.